src/Humanizer/Localisation/Formatters/DefaultFormatter.cs

```diff
@@ -67,7 +67,7 @@ public virtual string TimeSpanHumanize_Zero()
         /// <exception cref="System.ArgumentOutOfRangeException">Is thrown when timeUnit is larger than TimeUnit.Week</exception>
         public virtual string TimeSpanHumanize(TimeUnit timeUnit, int unit)
         {
-            if (timeUnit > TimeUnit.Week)
+            if (timeUnit > TimeUnit.Year)
```

Comment:
No, not anymore. I changed it when I began working and then I forgot to remove it. 
Thank you for pointing it out!